app/uk/gov/hmrc/hec/services/TaxCheckService.scala

```diff
@@ -68,12 +71,14 @@ class TaxCheckServiceImpl @Inject() (
     config.get[FiniteDuration]("hec-tax-check.expires-after").value.toDays
 
   def saveTaxCheck(
-    taxCheckData: HECTaxCheckData
+    taxCheckData: HECTaxCheckData,
+    fileCorrelationId: Option[UUID] = None
```

Comment:
I don't think we need to pass inthe `fileCorrelationId` here - it'll always be `None` for a fresh one coming in. When we update, we'd be calling `updateAllHecTaxCheck` I think

Reply:
yes, will remove it